### ISSUE TITLE
Line numbers won't be printed when printing folder

### ISSUE THREAD

**Issue by kuainx** [PDConSec/vsc-print#231]
I right-click the folder in Explorer in vsc and click the Print button, and Line Numbers doesn't show.
I tried to set Print: Line Numbers with on or inherit, it can control the single file line numbers, but can't work with folder.
thank you
![微信截图_20230522153328](https://github.com/PDConSec/vsc-print/assets/22209650/fc94e240-95f8-4f34-8b59-c675ebf1a836)

**Comment by PeterWone** [comment]
You are correct. Line numbering is not supported when multiple files are merge into a single print job. This is not a bug, it is a feature request. On that basis, I will move it to the Ideas discussion.
